src/common/dcgraph.cpp

```diff
@@ -452,7 +452,7 @@ void wxGCDCImpl::DoSetDeviceClippingRegion( const wxRegion &region )
 void wxGCDCImpl::DestroyClippingRegion()
 {
     m_graphicContext->ResetClip();
-#if !defined(__WXOSX__) && !defined(__WXMSW__) && !defined(__WXGTK__)
+#if !defined(__WXOSX__) && !defined(__WXMSW__) && !defined(__WXGTK__) && !defined(__WXQT__)
```

Comment:
Do we really need this code? AFAICS the only still used port not tested here is wxX11 but it has so many other problems that I'm not sure it's worth to cater to it specifically here.

Reply:
Done.